src/containers/shared/components/Transaction/OfferCreate/OfferCreateTableDetail.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Amount } from '../../Amount';
+
+export const OfferCreateTableDetail = (props: any) => {
+  const { t } = useTranslation();
+  const { instructions } = props;
+  const { gets, pays, price, pair, cancel } = instructions;
+
+  return pays && gets ? (
+    <div className="offercreate">
+      <div className="price">
+        <span className="label"> {t('price')}:</span>
+        <span className="amount">
+          {` ${Number(price)} `}
+          {pair}
+        </span>
+      </div>
+      <div className="amounts">
+        <span className="label">{t('buy')}</span>
+        <Amount value={gets} />
+        <span className="label">{`- ${t('sell')}`}</span>
+        <Amount value={pays} />
+      </div>
+      {cancel && (
+        <div className="cancel">
+          <span className="label">{t('cancel_offer')}</span>
+          {` #`}
+          <span className="sequence">{cancel}</span>
+        </div>
+      )}
+    </div>
+  ) : null;
```

Comment:
you don't need this ternary in this case, i believe. you can just do:

```javascript
return pays && gets && (
   <div>
      ...
   </div>
)
```

Reply:
I just tried that.  When that happens I get errors thrown in the tests that say nothing was returned.  If I do it the suggested way it will return `undefined`.  I guess React hates that.

Reply:
Maybe it'll be a bit neater with something like this:
```
return pays && gets && (
  ...
) || null
```